test-schema.json

```diff
@@ -27,6 +27,26 @@
                 "type": "array",
                 "items": { "$ref": "#/$defs/test" },
                 "minItems": 1
+            },
+            "specification":{
+                "description": "A reference to a specification document which defines the behavior tested by this test case. Typically this should be a JSON Schema specification document, though in cases where the JSON Schema specification points to another RFC it should contain *both* the portion of the JSON Schema specification which indicates what RFC (and section) to follow as *well* as information on where in that specification the behavior is specified.",
+
+                "type": "array",
+                "minItems": 1,
+                "items":{
+                    "properties": {
+                        "core": { "pattern": "^[0-9a-zA-Z]+(\\.[0-9a-zA-Z]+)*$", "type":"string" },
+                        "validation": { "pattern": "^[0-9a-zA-Z]+(\\.[0-9a-zA-Z]+)*$", "type":"string" },
+                        "ecma262": { "pattern": "^[0-9a-zA-Z]+(\\.[0-9a-zA-Z]+)*$", "type":"string" },
+                        "perl5": { "type":"string" },
+                        "quote": { "type":"string" }
+                    },
+                    "patternProperties": {
+                        "^rfc\\d+$": { "pattern": "^[0-9a-zA-Z]+.([0-9a-zA-Z]+)*$", "type":"string" }
+                    },
+                    "additionalProperties": false,
```

Comment:
Are ALL additional spec references going to be RFCs?  What about ISO?  We should put a `type: string` subschema under `additionalProperties` to just apply to anything else.